### PR TITLE
Refactor use commands next

### PR DESCRIPTION
**Pull request by fireflamemm**
Ok, so before you merge this in, you should probably pull down this branch and run the bot. Test every command you have. Because, I'm not certain that everything will work as I anticipate. For example, that's the first time that I have ever done Dependency Injection the way I did.

### CODE REVIEW

File: WatchinWeebsBot/obj/WatchinWeebsBot.csproj.nuget.dgspec.json [hunk 1]
```diff
@@ -1,23 +1,20 @@
 {
   "format": 1,
   "restore": {
-    "C:\\Users\\nourd\\source\\repos\\WatchinWeebsBot\\WatchinWeebsBot\\WatchinWeebsBot.csproj": {}
+    "C:\\Users\\johns\\source\\repos\\WatchinWeebsBot\\WatchinWeebsBot\\WatchinWeebsBot.csproj": {}
```
Comment (fireflamemm) [inline WatchinWeebsBot/obj/WatchinWeebsBot.csproj.nuget.dgspec.json:4]: I'm not sure why these changes were made. They look pretty sketch though.

**Comment by Klivess** [comment]
gg
